Modelica/Fluid/Machines.mo

```diff
@@ -425,20 +535,26 @@ Then the model can be replaced with a Pump with rotational shaft or with a Presc
             final state_in = Medium.setState_phX(port_a.p, inStream(port_a.h_outflow), inStream(port_a.Xi_outflow)),
             final state = medium.state) "Monitoring model"
        annotation (Placement(transformation(extent={{-64,-42},{-20,0}})));
-    protected
+  protected
     constant SI.Position unitHead = 1;
     constant SI.MassFlowRate unitMassFlowRate = 1;
-
+    parameter Boolean ignoreN = false "= true to ignore N of pump; only use if checkValve=false and use_powerCharacteristic=false"
```

Comment:
```suggestion
    parameter Boolean ignoreN = false "= true, if N of pump is ignored, i.e. N = N_nominal"
```

Reply:
"only use if checkValve=false and use_powerCharacteristic=false" - we don't use such description to comment annotation "enable" in other models

Reply:
Generally I agree, but I wanted to make an exception here.

Normally if you have enable=animation for an animation-parameter you can intuitively understand the logic and having the explanation would just be silly, but here enable is based on two different parameters that are on different tabs (and I don't want to move them) - and people will basically have to read the source to understand it.